webpack.config.js

```diff
@@ -361,4 +304,56 @@ module.exports = (env = process.env.NODE_ENV || 'development') => {
     },
     devtool,
   };
+
+  const mainConfig = {...baseConfig};
```

Comment:
This seems a bit hacky - open to suggestions on how to better make two configs from the same base.

Reply:
This is totally fine/quite idiomatic! The only thing I would change is just constructing each variant of the config in a single literal, e.g.:

```js
const mainConfig = {
  ...baseConfig,
  name: 'main',
  entry: isTest
    ? undefined
    : {
        inline: 'first-input-delay',
        main: './application.js',
      },
  //...
};
```